File: 07_javanetwork/src/main/java/net09/webserver/file/Test01.java
package net09.webserver.file;

import java.io.PrintWriter;
import java.net.ServerSocket;
import java.net.Socket;

import server.FileUtil;
import server.HttpServletRequest;
import server.HttpServletResponse;

public class Test01 {
	public static void main(String[] args) {
		try {
			ServerSocket server = new ServerSocket(8000);
			while (true) {
				Socket client = server.accept();
				HttpServletRequest request = new HttpServletRequest(
						client.getInputStream()
				); 
				HttpServletResponse response = new HttpServletResponse(
						client.getOutputStream()
				);
				
				response.setContentType("text/html; charset=utf-8");
				PrintWriter out = response.getWriter();
				out.println(FileUtil.readFile("WebContent/html/test.html"));
				out.close();
				
				response.send();
				
				client.close();
			}
		} catch (Exception e) {
			e.printStackTrace();
		}
	}
}

File: 16_node/04_server/test01.js
const http = require("http");
const port = 10001;

// 서버 생성
const server = http.createServer();

// 사용자가 접속했을때 실행
server.on("request", function () {
    console.log("사용자 요청 들어옴");
});

// 10001 번으로 구동
server.listen(port);

console.log(`${port}번으로 구동중..`);